docs/game_sdk/Store.md

```diff
@@ -0,0 +1,454 @@
+-H "Accept: application/json"
+
+// Returns
+
+{
+  [
+    {
+      "user_id": "53908232506183680",
+      "sku_id": "53908232599983680",
+      "application_id": "461618159171141643",
+      "id": "53908232506183999",
+      "type": 3
+    }
+  ]
+}
+```
+
+## Get Entitlement % GET /applications/{application.id#DOCS_GAME_SDK_SDK_STARTER_GUIDE/get-set-up}/entitlements/{entitlement.id#DOCS_GAME_SDK_STORE/data-models-entitlement-struct}
+
+Fetch an entitlement by its ID. This may be useful in confirming that a user has a given entitlement that another call or the SDK says they do.
+
+```
+curl https://discordapp.com/api/v6/applications/461618159171141643/entitlements/53908232506183999 \
+-H "Authorization: Bearer <token>" \
+-H "Accept: application/json"
+
+// Returns
+
+{
+  "user_id": "53908232506183680",
+  "sku_id": "53908232599983680",
+  "application_id": "461618159171141643",
+  "id": "53908232506183999",
+  "type": 3
+}
+```
+
+## Get SKUs % GET /store/applications/{application.id#DOCS_GAME_SDK_SDK_STARTER_GUIDE/get-set-up}/skus
+
+Get published SKUs for the given application id. You can filter the type of SKUs returned via the optional query parameter.
+
+###### Query Parameters
+
+| name      | value           | description                   |
+| --------- | --------------- | ----------------------------- |
+| sku_types | Discord.SkuType | the type of SKU to filter for |
+
+###### Example
+
+```
+curl https://discordapp.com/api/v6/store/applications/461618159171141643/skus?sku_types=1 \
+-H "Authorization: Bearer <token>" \
+-H "Accept: application/json"
+
+// Returns
+
+{
+  [
+    {
+      "id": "53908232599983680",
+      "type": 1,
+      "dependent_sku_id": null,
+      "application_id": "461618159171141643",
+      "manifest_labels": ["461618159171111111"],
+      "name": "My Awesome Test Game",
+      "access_type": 1,
+      "features": [1, 2, 3],
+      "system_requirements": {},
+      "content_ratings": {},
+      "release_date": "1999-01-01",
+      "legal_notice": {},
+      "price_tier": 1099,
+      "price": {},
+      "premium": false,
+      "locales": ["en-US"],
+      "bundled_skus": null
+    }
+  ]
+}
+```
+
+## Consume SKU % POST /store/applications/{application.id#DOCS_GAME_SDK_SDK_STARTER_GUIDE/get-set-up}/entitlements/{entitlement.id#DOCS_GAME_SDK_STORE/data-models-entitlement-struct}/consume
+
+Marks a given entitlement for the user as consumed, meaning it will no longer be returned in an entitlements check. **Ensure the user was granted whatever items the entitlement was for before consuming it!**
+
+###### Example
+
+```
+curl -X POST https://discordapp.com/api/v6/applications/461618159171141643/entitlements/53908232506183999/consume \
+-H "Authorization: Bearer <token>" \
+-H "Accept: application/json"
+
+// Returns
+
+{
+  "code": 0,
+  "message": "200: OK"
+}
+```
+
+## Get User Entitlements % GET /users/@me/applications/{application.id#DOCS_GAME_SDK_SDK_STARTER_GUIDE/get-set-up}/entitlements
```

Comment:
Sorry if it wasn't clear, this should not be used by game devs on their server. They can use the other endpoint to perform this action. This is not an OAuth enabled endpoint!